src/components/CommentCard.jsx

```diff
@@ -0,0 +1,46 @@
+
+import { useState, useEffect } from "react";
+import { getAllComments } from "../api";
+import { Axios } from "axios";
+import SingleReview from '../components/SingleReview'
+import { formatDate } from "../utils";
+
+
+const CommentCard = ({review, commentSection, setCommentSection, commentShow}) => {
+    
+
+    useEffect(() => {
+        getAllComments(review).then(({ comments } ) => {
+       
+           setCommentSection(comments)
+       }) 
+    }, [commentShow])
+
+    if (commentShow) {
+
+       return(
+        <section className="commentBox">
+       <ul className="commentList">
+        <h4>Comments: </h4>
+        {commentSection.map((comment) => {
+            const formattedDate = formatDate(comment.comment_id)
+           
+            return <li key={comment.comment_id} className="commentListItem">
+                <p>by {comment.author} at {formattedDate}</p>
```

Comment:
Nice mapping of the comments. 

Could you: 

- [ ] Add a message for any article that doesnt have any comments? Something like `no comments to display`